doc/tutorials/cpu_sharing.rst

```diff
@@ -41,36 +41,36 @@ Here is an example for affinity:
 Thread object state
 *******************
 
-There are three states for thread object, RUNNING, RUNNABLE and BLOCK. States transition happens in certain conditions.
+The thread object contains three states: RUNNING, RUNNABLE, and BLOCK.
 
 .. figure:: images/cpu_sharing_state.png
    :align: center
 
-After a new vCPU is created, the corresponding thread object is initiated. The vcpu layer will invoke a wakeup operation. After wakeup, the state for the new thread object is set to RUNNABLE, then follow its algorithm to determine to preempt the current running thread object or not. If yes, it will turn to RUNNING state. In RUNNING state, the thread object may turn back to RUNNABLE state when it runs out of its timeslice, or it yields the pCPU by itself, or be preempted. The thread object under RUNNING state may trigger sleep to transfer to BLOCKED state.
+After a new vCPU is created, the corresponding thread object is initiated. The vCPU layer invokes a wakeup operation. After wakeup, the state for the new thread object is set to RUNNABLE, and then follows its algorithm to determine whether or not to preempt the current running thread object. If yes, it turns to the RUNNING state. In RUNNING state, the thread object may turn back to the RUNNABLE state when it runs out of its timeslice, or it might yield the pCPU by itself, or be preempted. The thread object under RUNNING state may trigger sleep to transfer to BLOCKED state.
```

Comment:
`BLOCK` was the term used above, which one is correct, `BLOCK` or `BLOCKED`?

(a quick inspection of the code tells me it's `BLOCKED` but please confirm)